Database/Table Scripts/CreateHolidayRatesTableScript.sql

```diff
@@ -0,0 +1,11 @@
+IF (EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'dbo' AND TABLE_NAME = 'HolidayRates'))
```

Comment:
For holiday's rates may be different - higher or lower.

This table holds holiday dates and rate we want to charge for specific services (foreign key)